app/views/moments/show.html.erb

```diff
@@ -43,16 +43,18 @@
   </div>
 <% end %>
 
-<% if @resources.any? %>
-<div class="smallMarginTop">
-  <div class="label"><%= label_tag t('moments.show.resources') %></div>
-  <ul>
-    <% @resources.take(3).each do |item|  %>
-      <li><%= link_to item['name'], item['link'] %></li>
-    <%end %>
-    <li><%= link_to  "#{t('load_more')}...", "/resources?#{@resources_tags}" %></li>
-  </ul>
-</div>
+<% if @moment.resource_recommendations? %>
+  <% if @resources.any? %>
```

Comment:
We should combine these conditions with `&&`.

Reply:
Thank you for the speedy feedback, @julianguyen!  I'll fix that right away.